patches/lua-tree.patch

```diff
@@ -161,21 +190,20 @@ index b1093a2..0afc200 100644
  
  local tostring = tostring
 diff --git a/lua-tree/share/lua/5.1/kong/db/schema/init.lua b/lua-tree/share/lua/5.1/kong/db/schema/init.lua
-index 625bb0c63..3c7126165 100644
+index 489a666..1c660ed 100644
 --- a/lua-tree/share/lua/5.1/kong/db/schema/init.lua
 +++ b/lua-tree/share/lua/5.1/kong/db/schema/init.lua
-@@ -1,16 +1,12 @@
+@@ -1,15 +1,12 @@
  local tablex       = require "pl.tablex"
  local pretty       = require "pl.pretty"
  local utils        = require "kong.tools.utils"
 -local cjson        = require "cjson"
--local vault        = require "kong.pdk.vault".new()
+-local new_tab      = require "table.new"
+-local is_reference = require "kong.pdk.vault".new().is_reference
+-
 +local cjson        = { array_mt = {} } --- TODO(hbagdi) XXX analyze the impact
-+local uuid         = require "go.uuid".generate
```

Comment:
Is the removal of `local uuid = require "go.uuid".generate` expected? I see the ["docstring" entry](https://github.com/Kong/goks/blob/398d1bf24772bf79d1caaa65430eadb1583f5a20/patches/lua-tree.patch#L46) has not been removed.

Reply:
It is expected. I did not modify this patch manually, it was all done after updating the Lua tree & just simply replacing the existing patches for the files.

I just forgot to remove the docstring comment from the patch file. I'll go ahead and do that, nice catch.

Reply:
Removed here:
https://github.com/Kong/goks/pull/27/commits/de4b1492de389f9341fb03484adbdd1a6650d0ab